Backpack/src/main/java/net/skyscanner/backpack/calendar/presenter/BpkCalendarController.kt

```diff
@@ -74,14 +91,26 @@ abstract class BpkCalendarController {
     return CalendarDay(year, month, day).date == CalendarDay.today().date
   }
 
-  fun updateSelection(range: CalendarRange) {
+  fun updateSelectionForRange(range: CalendarRange) {
```

Comment:
Can we keep just one function here too `updateSelection(selection: CalendarSelection)`? So it can be called like this `updateSelection(CalendarRange())` or `updateSelection(CalendarDay())`

Reply:
I am fine with the change. Notice these methods are only used for testing at the moment.

Reply:
It's also used in the RN bridge we have, so changing the name will break it now.

Reply:
But keeping the name while changing the signature will be breaking anyway.
I am going to change it no issue.